src/store/modules/ui.store.js

```diff
@@ -228,6 +233,26 @@ export default {
         noFeatureInfo(state) {
             return state.featureInfoPosition === FeatureInfoPositions.NONE
         },
+        /**
+         * Flag to display to display give feedback button/form. On localhost, it will always shown
+         * for testing purpose
+         */
+        hasGiveFeedbackButton(state) {
+            if (state.hostname.includes('localhost')) {
+                return true
+            }
```

Comment:
Instead of only showing that on `localhost` it might be better to check that the value of `.hasDevSiteWarning` is true. So that on all our dev hosting both link will be visible (but will be different on either prod staging)